toml/src/test/java/com/electronwill/nightconfig/toml/TomlWriterTest.java

```diff
@@ -106,10 +108,48 @@ public void correctNewlinesMixed() {
 		TomlWriter tWriter = new TomlWriter();
 		String written = tWriter.writeToString(conf);
 		System.out.println(written);
-		assertLinesMatch(Arrays.asList("simple = 123", "", "[table]", "\tkey = \"value\"", ""),
+		assertLinesMatch(Arrays.asList("simple = 123", "", "[table]", "\tkey = \"value\"", "", ""),
```

Comment:
this additional line is not ideal but that's good enough, we can figure out a better solution later